packages/web3-common/src/types.ts

```diff
@@ -0,0 +1,37 @@
+export type JSONRPCId = string | number | null;
```

Comment:
Nit picky, but `JSONRPC` should be `JsonRpc` for both readability and to match Lodestar